test/experimental/key_error_test.rb

```diff
@@ -12,4 +12,37 @@ def test_corrects_hash_key_name
     assert_correction %("foo"), error.corrections
     assert_match %(Did you mean?  "foo"), error.to_s
   end
+
+  if KeyError.method_defined?(:receiver) && KeyError.method_defined?(:key)
```

Comment:
I think we could always run these tests since Ruby 2.5.0 (and later) is the only version DYM 1.2.0 is going to be compatible with.